dist/bitbucket/RoboFile.php

```diff
@@ -220,7 +219,7 @@ protected function runCypressTests()
       ->copy('.cypress/cypress.json', 'cypress.json', $force)
       ->copy('.cypress/package.json', 'package.json', $force);
     $tasks[] = $this->taskExec('sleep 30s');
-    $tasks[] = $this->taskExec('npm install cypress --save-dev');
+    $tasks[] = $this->taskExec('npm install cypress@9 --save-dev');
```

Comment:
Cypress released a major new version in May 2022 which breaks compatibility, so fixing version.